src/services/io/podio/JEventProcessorPODIO.cc

```diff
@@ -236,7 +236,7 @@ JEventProcessorPODIO::JEventProcessorPODIO() {
     "B0TrackerCKFTracksUnfiltered",
     "B0TrackerCKFTrackUnfilteredAssociations",
 
-    "CentralAndB0TrackerVertices",
+    "CentralAndB0TrackVertices",
```

Comment:
My original idea was that we could cut down on naming discrepancies according to https://github.com/eic/EICrecon/pull/1744#discussion_r2114936014
we could then rename `CentralTrackVertices` later.

I'm fine with doing this revert too.

Reply:
I thought you were agreed with this since you approved. I don't care about what naming. Just needs to be consistent.